test/BindingConvertersTests.ts

```diff
@@ -19,6 +19,21 @@ describe('Binding Converters', () => {
         },
         "$request": {
             string: "Https://mock/"
+        },
+        "falsyZero": {
+            string : "0",
+        },
+        "falsyFalse": {
+           string : "false",
+        },
+        "falsyNull": {
+            string : "null"
+        },
+        "falsyEmptyString": {
+           string : "",
+        },
+        "falsyUndefined": {
+          string : undefined
```

Comment:
Can we do these tests for the durable bindings as well?

Reply:
Let's do it in a separate PR as it not related to the change. 

Do we want to release the change for both v3 and v4 or only v4?
https://github.com/Azure/azure-functions-nodejs-worker/commit/896e88c44a71a07572fa6196ec1a09245f3e9262
It looks like it was not released for v3 yet

Reply:
Regarding adding tests in a new PR: That's fair, let's just make sure to create a tracking ticket for it then :) .

As for `896e88c`, is there a reason it hasn't been released yet for V3?

Reply:
no, I think it safe to release it to v3. I just saw that it was not released for v3 yet.

Reply:
In that case, let's release it to V3 as well :)